install_mac_requirements.sh

```diff
@@ -129,43 +138,43 @@ if ! brew help >/dev/null 2>&1 || [[ "$PATH" != *"$(brew --prefix)/bin"* ]]; the
     
     if $echoOn; then
         printf "> if [ -d \"/opt/homebrew\" ]; then\n"
-        printf ">     brew_prefix=\"/opt/homebrew\"\n"
+        printf ">     brewPrefix=\"/opt/homebrew\"\n"
```

Comment:
I prefer one of  brew_prefix, BREW_PREFIX, BREWPREFIX. This is more aligned with the "culture" of shell scripts than the camelCase conventions. Please adjust this.